ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/phase0/statetransition/epoch/RewardsAndPenaltiesCalculatorPhase0.java

```diff
@@ -30,6 +32,7 @@
 import tech.pegasys.teku.spec.logic.common.statetransition.epoch.status.ValidatorStatuses;
 
 public class RewardsAndPenaltiesCalculatorPhase0 extends RewardsAndPenaltiesCalculator {
```

Comment:
I think the log message would make more sense on AbstractEpochProcessor#processEpoch. We could check at the end of the processEpoch method.

Having it deep into the reward calculation isn't very intuitive and having it in the Phase0 implementation means that in the future, if someone overrides the method with Deneb, for example, we would lose this functionality